python/metric_pipeline_tasks/MatchedCatalogMeasureTasks.py

```diff
@@ -204,6 +204,51 @@ def run(self, matchedCatalog, metric_name):
                           afPercentile.value)*u.marcsec))
 
 
+def isSorted(l):
+    return all(l[i] <= l[i+1] for i in range(len(l)-1))
+
+
+def bins(window, n):
+    delta = window/n
+    return [i*delta for i in range(n+1)]
+
+
+class AMxWithHistTaskConfig(AMxTaskConfig):
+    bins = ListField(doc="Bins for histogram.",
+                     dtype=float, minLength=2, maxLength=1500,
+                     listCheck=isSorted, default=bins(30, 200))
+
+
+class AMxWithHistTask(Task):
```

Comment:
This run method is almost identical to the run method in the AMxTask, with the difference being the addition of the `extras` here. If this method is intended mainly as a demonstration, I think this is OK, but we might want to consolidate in the future if we begin using the `extras` more regularly.

Reply:
I have taken this suggestion and renamed the AMxWithHistTask to AMxTask.  This means that `extras` are carried around with every `AMx` measurement.